doc.go

```diff
@@ -35,5 +35,53 @@ increasing compile times or binary sizes, and avoiding any side effects from
 
 For sample applications and a tutorial, see the samples directory
 (https://github.com/google/go-cloud/tree/master/samples).
+
+
+Escaping the abstraction
+
+It is not feasible or desirable for APIs like blob.Bucket to encompass the full
+functionality of every provider. Rather, we intend to provide a subset of the
+most commonly used functionality. There will be cases where a developer wants
+to access provider-specific functionality, which might consist of:
+
+1. Top-level APIs. For example, blob does not expose a Copy, but some provider
```

Comment:
WDYT about dropping this list, or collapsing it into a parenthetical? Seems like overkill for here.

Reply:
Good points